test cases/failing/123 pkgconfig not relocatable outside prefix/meson.build

```diff
@@ -0,0 +1,15 @@
+project(
+  'pkgconfig-not-relocatable-outside-prefix', 'c',
```

Comment:
As we aren't building anything anymore, we can avoid declaring that we need the C language, and skip doing a compiler lookup/sanity check.

Reply:
Removed in the latest push.